assets/js/session/index.js

```diff
@@ -318,6 +318,8 @@ function handleDocumentKeyDown(hook, event) {
       insertCellAboveFocused(hook, "markdown");
     } else if (keyBuffer.tryMatch(["S"])) {
       addSection(hook);
+    } else if (keyBuffer.tryMatch(["c"])) {
+      captureCodeSnippet(hook);
     }
```

Comment:
In this specific case I'd stick to the button option. I'm all in for shortcuts, but this is probably not frequent enough to get a shortcut (especially that `c` is quite prominent and may be of use in the future).

Reply:
Yeah, most would probably use the button anyway.

Reply:
I do think having the option there would be handy for power-users but there is no evidence that would be the case so let's remove it for now.